arviz/plots/plot_utils.py

```diff
@@ -627,3 +630,41 @@ def filter_plotters_list(plotters, plot_kind):
         )
         return plotters[:max_plots]
     return plotters
+
+
+def get_plotting_method(plot_name, plot_module, backend):
+    """Returns plotting function for correct backend"""
+    _backend = {
+        "mpl": "matplotlib",
+        "bokeh": "bokeh",
+        "matplotlib": "matplotlib",
+        None: "matplotlib",
```

Comment:
I think that if backend is `None` the value should be the one stored in `rcParams["plot.backend"]`. For the time being while bokeh is experimental we can add a warning in rcParams when setting bokeh as default backend.